tools/cli/isolation_groups_test.go

```diff
@@ -347,3 +332,217 @@ func TestAdminUpdateGlobalIsolationGroups(t *testing.T) {
 		})
 	}
 }
+
+func TestAdminGetDomainIsolationGroups(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
```

Comment:
this line is not needed on new versions of gomock. you can remove everywhere you see